src/api/app/views/webui/package/_basic_info.html.haml

```diff
@@ -26,5 +26,8 @@
   $('#cancel-in-place-editing').on('click', function () {
     $('.in-place-editing .basic-info').toggleClass('d-none');
     $('.in-place-editing .editing-form').toggleClass('d-none');
+    $('#package_details_title').val("#{escape_javascript(package.title)}");
+    $('#package_details_url').val("#{escape_javascript(package.url)}");
+    $('#package_details_description').val("#{escape_javascript(package.description)}");
```

Comment:
Hm. What about making use of a reset button instead of custom javascript? :) 

https://www.w3schools.com/tags/att_input_type_reset.asp

Reply:
@hennevogel thanks for the advice. We have tried the reset and it's working as expected. :raised_hands: